middleware/markdown/renderer.go

```diff
@@ -1,139 +0,0 @@
-package markdown
-
-import (
-	"bytes"
-)
-
-// SummaryRenderer represents a summary renderer.
-type SummaryRenderer struct{}
```

Comment:
So with this gone, how do we render summaries of a post? This is how we render https://caddyserver.com/blog - all the snippets under the titles are through this type. We render only content elements, without formatting.